src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -48,38 +49,44 @@
                 abstract_arg('argument value resolvers'),
             ])
 
+        ->set('argument_resolver.specified_resolver', SpecifiedResolverValueResolver::class)
+            ->args([
+                tagged_locator('controller.argument_value_resolver', 'name'),
```

Comment:
Should the service exclude itself here or is it done automatically?

Reply:
this should be done by default since #48685